app/views/posts/index.html.erb

```diff
@@ -1,2 +1,27 @@
+<% content_for :title, "#{@user.name}'s Posts" %>
 <h1>Posts#index</h1>
-<p>This is the list of all POSTS. Your are now at 'posts/'</p>
\ No newline at end of file
+<section class = "main-container">
+  <%= render "users/user", user: @user %>
+  <% @user.posts.each do | post | %>
+    <%= link_to user_post_path(@user, post) do %>
+      <section class = "post-comments">
+      <%= render "post", post: post %>
+      
+      <div class="user-post-comments">
+        <h2>
+          <span>Comments for Post: </span>
+          <span class="green-text"><%="#{post.title}"%> </span>
+        </h2>
+        <% post.recent_five.each do |comment| %>
+          <div class="post-comment">
+            <p>
+              <span><strong><%= comment.author.name %>:</strong></span>
+              <span><%= comment.text %></span>
+            </p>
+          </div>
+        <% end %>
+      </div>
+      </section>
+    <% end %>
+  <% end %>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update the above selected code in the way it shows a message when a post doesn't have any comments. This can let users know if they can add a new comment. i.e.
```ruby
....
  <% @user.posts.each do | post | %>
    <%= link_to user_post_path(@user, post) do %>
      <section class = "post-comments">
      <%= render "post", post: post %>
      
      <div class="user-post-comments">
        <h2>
          <span>Comments for Post: </span>
          <span class="green-text"><%="#{post.title}"%> </span>
        </h2>
        <% if post.comments_counter > 0 %>
          <% post.recent_five.each do |comment| %>
            <div class="post-comment">
              <p>
                <span><strong><%= comment.author.name %>:</strong></span>
                <span><%= comment.text %></span>
              </p>
            </div>
          <% end %>
        <% else %>
          <span><%= "No comments for this post, Add a comment" %></span>
        <% end %>
      </div>
      </section>
    <% end %>
  <% end %>
  ....

```